### DISCUSSION TITLE
Mob Programming RPG Game Integration

### DISCUSSION THREAD

**Discussion by mrozbarry** [mobtimeapp/mobtime#264] [Ideas]
Related threads:
 - [Eddie Bush's tweet](https://twitter.com/eddiebush/status/1492934322432757764)
 - [Mob Programming RPG](https://github.com/willemlarsen/mobprogrammingrpg)

So it sounds like there should be a bot that can receive input from players to check off XP-related tasks, and allow a given player to advance through levels.

**Comment by theparticleman** [comment]
I wouldn't want the RPG elements to be a required part of using the timer, but as an optional feature it sounds great!

**Reply by mrozbarry** [reply]
Integrations will be opt-in, will be separate, and may not always be free (but maybe some will 🤷 ). Everyone will have access to free basic timers forever.

I'm still working out details about how people will initialize and use these integrations, but they will activated/purchased/configured using a separate control panel.  If you're interested in discussing integrations in general, I started [another discussion thread](https://github.com/mobtimeapp/mobtime/discussions/265) about that.

**Reply by Bogghead** [reply]
Alex, are you familiar with Dillon Kearns' [integration with the mobster timer](https://github.com/dillonkearns/mobster#rpg-mode)? The mechanism for when and how to prompt each mob member on how they did in completing the role's task is worked out there, or at least one implementation is there.

**Reply by mrozbarry** [reply]
I'm not familiar, but I think that's okay, I'd rather go in with a fresh take. I'm sure, simply by the nature of how the mob rpg is played, there will be similarities, though.
